### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by ahzamir**
In this branch I have worked on the desktop version:

- I added the CSS for all the pages responsive to the desktop version.
- I added some HTML tag class changes required for the desktop version styling.
- I added some new tags and elements that needed to be in the desktop version.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -407,3 +427,326 @@ ul {
...
+    padding: 0%;
+  }
+
+  .project1 {
+    background: url(images/backgroundimg1.svg);
+  }
+
+  .project2 {
+    background: url(images/backgroundimg2.svg);
+  }
```
Comment (Amrhub) [inline style.css]: - [x] Let's kindly make sure our background image doesn't repeat to match figma design and we can make it cover the entire card
```css
background-size: cover;
background-repeat: no-repeat;
```
Your implementation | Figma Design
:------:|:-----:
image | image

File: style.css [hunk 2]
```diff
@@ -407,3 +427,326 @@ ul {
...
+    background: #ff6b00;
+    font-family: 'Inter', sans-serif;
+    margin-top: 377px;
+    height: 58px;
+    padding: 0%;
+  }
+
+  .project1 {
+    background: url(images/backgroundimg1.svg);
+  }
```
Comment (Amrhub) [inline style.css]: - [x] I know it's kind of confusing but if you pay closer attention in Figma design project-1 is like the rest of the projects but it has a hover-effect where when you hover over it the dark-background color is removed and title, description, and technologies is also removed
then you are left with `See Project`

File: style.css [hunk 3]
```diff
@@ -407,3 +427,326 @@ ul {
...
+  .see-project2 {
+    border: 1px solid #ff6b00;
+    background: #ff6b00;
+    font-family: 'Inter', sans-serif;
+    margin-top: 377px;
+    height: 58px;
+    padding: 0%;
+  }
+
+  .project1 {
```
Comment (Amrhub) [inline style.css]: - [x]  Currently our paragraph `project-description` is aligned center let's kindly fix that and make it align to the left to match the Figma design 

Your implementation | Figma Design
:------:|:-----:
image | image

**Review by Amrhub — CHANGES_REQUESTED** [review]
Hi @ahzamir,  👋🏻 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!
These changes aim to add some awesomeness to your awesome project and code skills.

## To Highlight ✔️ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -43,104 +51,184 @@ <h2 class="hsecond">I am a Software Developer</h2>
     </section>
     <section class="recent-work">
       <h2 class="title-work">My Recent Work</h2>
```
Comment (thecodechaser) [inline index.html:53]: - [ ] As requested by the previous reviewer when we hover on any project the content of the card should be replaced with the `see project` button. Please fix this issue and make sure it matches the design on the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)template.

#### See this image of Figma design, The mouse hovers on the first project and we can see it's showing the see project button and the content of the card has been changed. If you still need more clarification please see this [project](https://devmahmoud.github.io/portfolio/).

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @ahzamir 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahzamir** [comment]
- @Amrhub @thecodechaser 
- Thank you for your reviews and feedback. As you requested when we hover on any project, the card's content should be replaced with the see project button and the background without gradient but as you know I have just one image without gradient and the other images are with the gradient in Figma so I will do the hover on my projects but I have to use the same image on every project background.

**Comment by thecodechaser** [comment]
> * @Amrhub @thecodechaser
> * Thank you for your reviews and feedback. As you requested when we hover on any project, the card's content should be replaced with the see project button and the background without gradient but as you know I have just one image without gradient and the other images are with the gradient in Figma so I will do the hover on my projects but I have to use the same image on every project background.

@ahzamir Hello, No Problem you can use the same gradient image for every project

**Review by ShadyShawkat — APPROVED** [review]
## Hi @ahzamir, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ahzamir** [comment]
@thecodechaser @ShadyShawkat @Amrhub 

- Thank you very much for your reviews and feedback

**Pull request merged**